src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +136,27 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testAesdefaultmodepadding() throws InvalidParametersException {
+        String msg = "Test AES Default";
+        String decrptedMsg;
+        byte[] decrypted;
+        decrptedMsg = StringHelper.encrypt(msg);
+
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.DECRYPT_MODE, sks);
+            decrypted = cipher.doFinal(StringHelper.hexStringToByteArray(decrptedMsg));
```

Comment:
@Neurrone  would be like this

   
      private static byte[] encryptWithoutSpecifyingAlgorithmParams(String value) throws 
      InvalidParametersException {
        byte[] encrypted;
        try {
            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
            Cipher cipher = Cipher.getInstance("AES");
            cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
            encrypted = cipher.doFinal(StringHelper.hexStringToByteArray(value));
            return encrypted;
        } catch (Exception e) {
            throw new RuntimeException(e);
        }

    }`
   
    private static byte[] encryptWithSpecifyingAlgorithmParams(String value) throws InvalidParametersException {
        byte[] encrypted;
        try {
            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
            Cipher cipher = Cipher.getInstance("AES/ECB/PKCS5Padding");
            cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
            encrypted = cipher.doFinal(StringHelper.hexStringToByteArray(value));
            return encrypted;
        } catch (Exception e) {
            throw new RuntimeException(e);
        }

    }

    @Test
    public void testDefaultModeAndPadding() throws InvalidParametersException {
        String msg1 = "AAAAAA";
        //136 bits and more
        String msg2 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
        String msg3 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCCAAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
        byte[] expectedCiphertext;
        byte[] actualCiphertext;
        String[] inputMsg = {msg1, msg2, msg3};
        for (String msg : inputMsg) {

            actualCiphertext = encryptWithoutSpecifyingAlgorithmParams(msg);
            expectedCiphertext = encryptWithSpecifyingAlgorithmParams(msg);
            assert Arrays.equals(actualCiphertext, expectedCiphertext);
        }
    }`